pkg/ts/recipe/RecipeParser.ts

```diff
@@ -7,17 +7,19 @@
  */
 
 import {logFactory} from '../utils/log.js';
+import {SlotSpec, RecipeSpec, Recipe, Container, ParticleSpec, ParticleId, Store, Slot} from './types.js';
```

Comment:
Also, please make 'Dictionary' come out of 'types.js'.

Reply:
Dictionary comes from `../utils/types.ts` or did you want it re-exported from `./types.ts'?

Reply:
Re-export, yes.